css-color/Overview.bs

```diff
@@ -2653,9 +2667,6 @@ Issue: The code below is only for sRGB, and duplicates the
 			Otherwise, set the channel's value to
 			<code>((channel + 0.055) / 1.055) ^ 2.4</code>.
 
-			Note: This reverses the logarithmic power scaling of the sRGB gamut,
-			so the value of the channel is  linear with respect to the amount of light produced.
```

Comment:
Pointing out that the value is linear wrt the light produced (i.e., it is additive) still seems valuable.